src/Components/Server/src/ComponentHub.cs

```diff
@@ -143,7 +165,7 @@ private async void CircuitHost_UnhandledException(object sender, UnhandledExcept
             try
             {
                 Log.UnhandledExceptionInCircuit(_logger, circuitId, (Exception)e.ExceptionObject);
-                await circuitHost.Client.SendAsync("JS.Error", e.ExceptionObject);
+                await NotifyClientError(circuitHost.Client, e.ExceptionObject.ToString());
```

Comment:
This bit is not important as I already changed this on a separate PR that has to make its way to master.